src/pages/costModels/costModelsDetails/components/markup.styles.ts

```diff
@@ -1,4 +1,5 @@
-import { global_FontSize_xl, global_spacer_md } from '@patternfly/react-tokens';
+import global_FontSize_xl from '@patternfly/react-tokens/dist/js/global_FontSize_xl';
```

Comment:
Also here, is this how we suppose to import?

Reply:
Yes. Please see the description above